data/en/mbs_0158.json

```diff
@@ -0,0 +1,610 @@
+                        ]
+                    },
+                    {
+                        "type": "Question",
+                        "id": "reporting-period-block-2",
+                        "title": "Reporting period",
+                        "questions": [{
+                            "type": "DateRange",
+                            "id": "reporting-period-question-2",
+                            "title": "What are the dates of the period that you will be reporting for?",
+                            "answers": [{
+                                    "type": "Date",
+                                    "id": "period-from",
+                                    "label": "From",
+                                    "mandatory": true,
+                                    "q_code": "11"
+                                },
+                                {
+                                    "type": "Date",
+                                    "id": "period-to",
+                                    "label": "To",
+                                    "mandatory": true,
+                                    "q_code": "12"
+                                }
+                            ]
+                        }]
+                    },
+                    {
+                        "id": "turnover-block",
+                        "type": "Question",
+                        "title": "Turnover",
+                        "questions": [{
+                            "type": "General",
+                            "title": "For the period {{format_conditional_date (answers['period-from'], exercise.start_date)}} to {{format_conditional_date (answers['period-to'], exercise.end_date)}}, what was the value of {{respondent.trad_as_or_ru_name}}’s <em>total turnover</em>, excluding VAT?",
+                            "guidance": {
+                                "content": [{
+                                        "title": "Include:",
+                                        "list": [
+                                            "exports",
+                                            "payments for work in progress",
+                                            "costs incurred and passed on to customers",
+                                            "income from sub-contracted activities",
+                                            "commission",
+                                            "sales of goods purchased for resale",
+                                            "revenue earned from other parts of the business not named (please supply at fair value)"
+                                        ]
+                                    },
+                                    {
+                                        "title": "Exclude:",
+                                        "list": [
+                                            "VAT",
+                                            "income from the sale of fixed capital assets",
```

Comment:
Income is a capital letter in spec

Reply:
This is very inconsistent with the rest of the content. Assume this is GSheets auto formatting

Reply:
Yeah it was a capital but I assumed it wasn’t supposed to be